### ISSUE TITLE
panic in go arm

### ISSUE THREAD

**Issue by naveensrinivasan** [google/safebrowsing#55]
I am trying to run the sblookup on arm.
```
echo "http://testsafebrowsing.appspot.com/apiv4/ANY_PLATFORM/MALWARE/URL/" | ./sblookup -apikey=<redacted>
safebrowsing: 2017/04/28 18:27:49 database.go:106: no database file specified
safebrowsing: 2017/04/28 18:28:05 database.go:336: database is now healthy
safebrowsing: 2017/04/28 18:28:05 safebrowser.go:504: Next update in 30m17s
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x4 pc=0x1265c]

goroutine 1 [running]:
sync/atomic.addUint64(0x1077cddc, 0x1, 0x0, 0x0, 0x0)
	/usr/local/go/src/sync/atomic/64bit_arm.go:31 +0x4c
github.com/google/safebrowsing.(*SafeBrowser).LookupURLs(0x1077cd80, 0x10745e78, 0x1, 0x1, 0x108ba4b0, 0x43, 0x0, 0x0, 0x0)
	/home/pi/go/src/github.com/google/safebrowsing/safebrowser.go:407 +0x3c8
main.main()
	/home/pi/go/src/github.com/google/safebrowsing/cmd/sblookup/main.go:95 +0x22c
```

Looks like an issue. Am I right? Is this not supported on arm?

**Comment by colonelxc** [comment]
I didn't test it on arm specifically, but I was able to reproduce the issue with GOARCH=386. #56 Resolves the issue there, and should for you as well. Re-open (or open a new issue) if this doesn't work after #56 is merged.
